### PR TITLE
OD-492 [Fix] Create chat settings for the LFD directory & featured list layout

### PR DESCRIPTION
**Pull request by skrupnyk**
@sofiiakvasnevska @inna-bieshulia 
## Issue
OD-492 https://weboo.atlassian.net/browse/OD-492

## Description
Added title 'Email field', fixed transition to chat

## Screenshots/screencasts
https://storyxpress.co/video/kqjj5ius2kawddgf5
https://storyxpress.co/video/kqjj6xuwt2iqxka73

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Comment by skrupnyk** [comment]
@tonytlwu this fix has been tested on staging

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/layout-javascript/agenda-code.js [hunk 1]
```diff
@@ -601,8 +601,6 @@ DynamicList.prototype.attachObservers = function() {
           return;
         }
 
-        _this.$container.find('.new-agenda-list-container, .dynamic-list-add-item').addClass('hidden');
```
Comment (tonytlwu) [inline js/layout-javascript/agenda-code.js:604]: What's this for and why is it necessary for this feature? I see there's a lot of code changes in all the layouts where you're showing/hiding elements on the page before an entry is opened. I don't understand why this is related to the requirements of this ticket.

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Rejected +1

File: js/layout-javascript/agenda-code.js (hunk 1 above)
Comment (skrupnyk) [reply js/layout-javascript/agenda-code.js:604]: I apologize for the delay. This function was not written by me and was bundled along with the previous changes. Would it be better to undo someone else's changes?

Comment (tonytlwu) [reply js/layout-javascript/agenda-code.js:604]: Image 2021-07-28 at 10 51 17 am

@skrupnyk If you go to the **Files changed** tab (see above), you can see all the code changes for this PR. You should know why every single line of the changes are necessary. If not, you need to find out/clarify/remove the changes.

**Comment by skrupnyk** [comment]
link to the new pull request https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/597

**Pull request closed without merging**